packages/hub/schema.graphql

```diff
@@ -594,6 +611,22 @@ type RelativeValuesPairCache implements Node {
   secondItem: String!
 }
 
+type SearchEdge {
+  cursor: String!
+  node: Searchable!
+  rank: Float!
+  slugSnippet: String!
+  textSnippet: String!
+}
+
+type Searchable implements Node {
+  id: ID!
+  link: String!
+  object: SearchableObject!
+}
```

Comment:
It's possible that there's more indirection here than necessary. I attach search query-related params to the edges (`rank`, `slugSnippet`, `textSnippet`), but then the found node is again not an object itself but a wrapper for the actual object that's a union.